### PR TITLE
Integrate views removed from d-r[-v]

### PR DESCRIPTION
**Pull request by njoyard**
A quick summary of the changes made:
- Gave names to API views because their default names conflicted with other view names
- Added tags to make group/chamber/country URLs (1)
- Moved views each to its own file and integrated views removed from d-r[-v]
- Integrated URLs removed from d-r[-v]
- Made URLs more homogenous ~~(allow singular & plural ; all URLs default to singular)~~ : moved /groups/ to /group/ and added 301 redirect from old to new

(1) I prefer using tags instead of using get_absolute_url. As models are defined in a dependency, they cannot "know" what the user application wants to use as the "canonical model url" (for example for groups it may be rep list filtered by this group, or a detail page for the group, or whatever). I could have made inherited models in memopol just to override get_absolute_url, but then making sure foreign keys use overriden models is not made easy AT ALL by django. It's possible, but the code is horrible.

### CODE REVIEW

**Comment by njoyard** [comment]
This is now ready for review, @jpic :)

File: templates/home.haml [hunk 1]
```diff
@@ -5,7 +5,7 @@
       %p
         Memopol is reachable only in <b>reduced functionality mode</b>.
         By the way, you could access to
-        <a href="{% url 'representatives:representative-list' %}">the list of MEPs</a>.
+        <a href="{% url 'representative-list' %}">the list of MEPs</a>.
```
Comment (jpic) [inline templates/home.haml]: Why did you remove namespaces ?

**Comment by jpic** [comment]
Now that it maintains old URLs with permanent redirects, this doesn't break anything to my knowledge at the time of writing.

As discussed on IRC, while this PR does mainly code moves, it's part of a larger plan, so please follow the usual CD workflow and proceed to deployment+monitoring when you're ready.

:guitar:

File: templates/home.haml (hunk 1 above)
Comment (njoyard) [inline templates/home.haml]: As discussed on IRC those views are in memopol now. We could prefix all views with 'memopol:' but that does not seem very useful.
